test/app-tap/msgpackffi.test.lua

```diff
@@ -117,12 +117,13 @@ end
 
 tap.test("msgpackffi", function(test)
     local serializer = require('msgpackffi')
-    test:plan(13)
+    test:plan(14)
     test:test("unsigned", common.test_unsigned, serializer)
     test:test("signed", common.test_signed, serializer)
     test:test("double", common.test_double, serializer)
     test:test("decimal", common.test_decimal, serializer)
     test:test("uuid", common.test_uuid, serializer)
+    test:test("datetime", common.test_datetime, serializer)
```

Comment:
Same string should be added to `test/app-tap/msgpack.test.lua`. Sorry i didn't notice it right away

Reply:
Looks like it was similarly forgotten to add calls for uuid and decimal (there are test_decimal and test_uuid called from msgpackffi.test.lu, but not from msgpack.test.lua).

What is the point to divide such strangely? What was purpose of introducing separate ffi test for msgpack here? Should they test the same set of primitive types?

Reply:
msgpackffi.test.lua checks encode/decode for msgpackffi module, and msgpack.test.lua checks encode/decode for msgpack module.
```lua
msgpack = require('msgpack')
x = msgpack.encode({1, 2, 3})
msgpack.decode(x)
---
- [1, 2, 3]
...
```
```lua
msgpackffi = require('msgpackffi')
x = msgpackffi.encode({1, 2, 3})
msgpackffi.decode(x)
---
- [1, 2, 3]
- 5
...
```

Reply:
I think it's better to check both modules.

Reply:
```diff
commit c6e188236ce93188d6f7c0177b70318cfe211f5a
Author: Timur Safin <tsafin@tarantool.org>
Date:   Wed Dec 15 17:44:38 2021 +0300

    temp - msgpack.test.lua

diff --git a/test/app-tap/msgpack.test.lua b/test/app-tap/msgpack.test.lua
index 6b75b6999..11efbb38b 100755
--- a/test/app-tap/msgpack.test.lua
+++ b/test/app-tap/msgpack.test.lua
@@ -265,13 +265,14 @@ end
 
 tap.test("msgpack", function(test)
     local serializer = require('msgpack')
-    test:plan(14)
+    test:plan(15)
     test:test("unsigned", common.test_unsigned, serializer)
     test:test("signed", common.test_signed, serializer)
     test:test("double", common.test_double, serializer)
     test:test("boolean", common.test_boolean, serializer)
     test:test("string", common.test_string, serializer)
     test:test("nil", common.test_nil, serializer)
+    test:test("datetime", common.test_datetime, serializer)
     test:test("table", common.test_table, serializer, is_array, is_map)
     test:test("ucdata", common.test_ucdata, serializer)
     test:test("depth", common.test_depth, serializer)
```

Reply:
In a longer run, if the sole difference between `msgpackffi.test.lua` and `msgpack.test.lua` are their different `require`s then we could commonize them in the single file, and parametrise function to include different modules...